package.json

```
@@ -98,7 +98,7 @@
     "dev": "npm run build -- --watch",
     "build:docs": "appium-docs build",
     "dev:docs": "appium-docs build --serve",
-    "publish:docs": "appium-docs build --deploy -b docs-site -m 'docs: auto-build docs for appium-xcuitest-driver@%s' --rebase",
+    "publish:docs": "appium-docs build --deploy -b docs-site -m 'docs: auto-build docs for appium-xcuitest-driver@%s' --rebase --remote origin --branch gh-pages --push",
```

there is some brokenness in appium-docs which I am about to open a PR for. when that lands, please update to it. this should then work:

```suggestion
    "publish:docs": "appium-docs build --deploy -b docs-site -m 'docs: auto-build docs for appium-xcuitest-driver@%s' --push",
```

`--rebase` now defaults to `true`, `--remote` now defaults to `origin`, and `--branch` now defaults to `gh-pages` (if this wasn't true previously).  the only thing here that does _not_ default to `true` is `--push`.

PRO TIP: when using `implies` in yargs, if the depended-upon arg is a `boolean` and the default is not `true`, _you must be explicit_ about it and set `default: false`.

---

@boneskull this suggestion is essentially a revert, should we just close this PR instead?